hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/ContinuousQueryAddListenerCodec.java

```diff
@@ -170,7 +170,7 @@ public void handle(ClientMessage clientMessage) {
         /**
          * @param events List of events in the form of data that holds the details of the event such as key, value, old value,
          *               new value and creation time.
-         * @param source Source that dispathces this batch event.
+         * @param source Source that dispatches this batch event.
```

Comment:
I see that there are still some codec files are left. Could you update them as well, as you did for the statistics codec?

Reply:
Sure. Sorry. I should have looked through the full list.